ra/ra.go

```diff
@@ -1667,12 +1693,6 @@ func (ra *RegistrationAuthorityImpl) NewOrder(ctx context.Context, req *rapb.New
 		Expires:        &expires,
 	}
 
-	// Check that the registration ID in question has rate limit space for another
-	// pending order
-	if err := ra.checkPendingOrderLimit(ctx, *req.RegistrationID); err != nil {
```

Comment:
Should we not also kill off the `checkPendingOrderLimit` code?

Reply:
Sure, that makes sense. I think I was overly generous in what I left around for the deployability guidelines.

Reply:
Resolved in 000d0c9